backend/zeno_backend/database/select.py

```diff
@@ -183,7 +183,7 @@ async def projects(
             Project(
                 uuid=res[0],
                 name=res[1],
-                view=match_instance_view(res[3]),
+                view="",
```

Comment:
This can break login for existing projects on the homescreen with bad view specs. Don't need view here anyway